### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by hussam789** [saeedranjbar12/mtlcfci#1]
Hey,
Do you provide training code with the loss functions?

**Comment by tvkamara** [comment]
I would also like to see the training code of this project. I am specially interested for the compressibility loss mentioned in the paper.
